addons/mail/static/src/messaging/widget/form_renderer/form_renderer_tests.js

```diff
@@ -96,6 +105,8 @@ QUnit.test('basic chatter rendering', async function (assert) {
 QUnit.test('basic chatter rendering without followers', async function (assert) {
     assert.expect(6);
 
+    // FIXME could be removed once task-2248306 is done
+    this.data['mail.activity'] = {fields: {}};
```

Comment:
why not in `beforeEach()` of `messaging.testUtils`?

Reply:
Was not sure it was good as it could be deleted (the model is never really used, but needed to fix views creations until mentioned task is merged)

Reply:
Given that info, you still think it should be in messaging.testUtils ?

Reply:
If we make activity tests that require RPC, then we will need `data['mail.activity']`, so it's still nice to move it to `messaging.testUtils:beforeEach()`, and avoid a `FIXME` comment at the same time